### ISSUE TITLE
Use cy.api() with blob file returning TypeError

### ISSUE THREAD

**Issue by CarolinaSL** [filiphric/cypress-plugin-api#102]
Hello! I'm with problems when I try to send a blob file through formData. The error returned is the following:

![image](https://user-images.githubusercontent.com/17781953/210276965-d5912e58-89fb-4d7e-b8d5-1f114e314099.png)

Some points : 

-The same error is obtained using requestMode. 

Cypress version: 12.1.0
Cypress-api-plugin version: 2.9.0

![image](https://user-images.githubusercontent.com/17781953/210277069-33fd3dc7-5c4f-47fa-9e55-c0aa44c3f9d5.png)

Here is the piece of code:

` .then((binary) => Cypress.Blob.binaryStringToBlob(binary))
        .then((blob) => {
            var path = pathFileFixture;
            let formData = new FormData();
            formData.append('file', blob, fileName);
            cy.api({
                method: 'POST',
                url: Cypress.env('api_url').file_storage,
                body: formData`

**Comment by filiphric** [comment]
will create support for this soon 👍 thanks for the report. one question - before `.then()` command, do you use `cy.fixture()` to load the file?

**Comment by CarolinaSL** [comment]
Yes, I use ```cy.fixture()``` to load the file. I appreciate your effort, thanks!

**Comment by piotrtar** [comment]
Does it matter if you import a file by 
`import fileName from '../../fixtures/fileName.json';` or cy.fixture()?

**Comment by filiphric** [comment]
it seems that the `formData` on its own is not the core of the problem. I have been able to work OK with the following test:
```js
it('uploads a file', () => {

  cy.fixture('logo.png', 'binary').then(image => {
    const blob = Cypress.Blob.binaryStringToBlob(image, 'image/png');
    const formData = new FormData();
    formData.append('image', blob, 'logo.png');

    cy.api({
      method: 'POST',
      url: '/',
      body: formData,
      headers: {
        'content-type': 'multipart/form-data'
      },
    })
  })


});
```
it doesn’t do too much, but it does not throw the error. is it possible for you to provide an example of what the response for such API call is?

**Comment by samirf-stubben-edge** [comment]
I'm facing the same issue and it seems to be related to the API response processing. Not the request.

Here is the response from my API:

> <html>
> <head><title>413 Request Entity Too Large</title></head>
> <body>
> <center><h1>413 Request Entity Too Large</h1></center>
> <hr><center>nginx</center>
> </body>
> </html>
> 
> 
> 
> 
> 
> 
> 
![Captura de tela 2023-06-14 095934](https://github.com/filiphric/cypress-plugin-api/assets/130555919/b911c6aa-1f56-4c6f-916c-4a4289c17fcb)



![Captura de tela 2023-06-14 095752](https://github.com/filiphric/cypress-plugin-api/assets/130555919/d03c4f70-5f2d-4dad-8a86-99c98af1aa43)

**Comment by SivAlka** [comment]
Seeing the same issue. I am using a fixture and modifying some objects in the json before passing to the endpoint. The operation itself seems successful as I can see the desired values changed in my application but I get the above mentioned error at the end. There is no error when using cypress native method

```
cy.fixture('').then((data) => {
    data.key1 = some value;
// written in commands.js
    cy.putWithTokenAPI(urlPath, accessToken, data).then((response) => {
      expect(response.status).to.eq(200);
    });
  });
```
